### PR TITLE
docs(readme): Update readme

### PR DESCRIPTION
**Pull request by lambtron**
- updated examples
- added more resources
- added youtube

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,34 +62,34 @@ and [releases](https://github.com/denoland/deno/releases) for other options.
...
+  return new Response("Hello, World!");
 }
 
-async function serve(conn: Deno.Conn) {
-  for await (const { respondWith } of Deno.serveHttp(conn)) {
-    respondWith(new Response("Hello world"));
-  }
-}
+console.log("Listening on http://localhost:8000");
+serve(handler);
```
Comment (dsherret) [inline README.md:81]: Maybe we should just inline the handler here?

```ts
serve((_req) => {
  return new Response("Hello, World!");
});
```

File: README.md [hunk 2]
```diff
@@ -62,34 +62,34 @@ and [releases](https://github.com/denoland/deno/releases) for other options.
...
 
-You can find a deeper introduction, examples, and environment setup guides in
-the [manual](https://deno.land/manual).
+[More examples](https://examples.deno.land/).
+
+### Additional Resources
 
-The complete API reference is available at the runtime
-[documentation](https://doc.deno.land).
+- **[The Deno Manual](https://deno.land/manual)** is a great starting point for [additional examples](https://deno.land/manual/examples), [setting up your environment](https://deno.land/manual/getting_started/setup_your_environment), [using npm](https://deno.land/manual/node), and more.
```
Comment (dsherret) [inline README.md:88]: Run `./tools/format.js`

**Review by dsherret — APPROVED** [review]
Very nice!

File: README.md (hunk 1 above)
Comment (ry) [reply README.md:81]: It can be simplified to a single line too

File: README.md [hunk 3]
```diff
@@ -1,22 +1,22 @@
 # Deno
 
-[![Build Status - Cirrus][]][Build status] [![Twitter handle][]][Twitter badge]
-[![Discord Chat](https://img.shields.io/discord/684898665143206084?logo=discord&style=social)](https://discord.gg/deno)
+[![Build Status - Cirrus][]][Build status] [![Twitter badge][]][Twitter link]
```
Comment (bartlomieju) [inline README.md:3]:
```suggestion
[![](https://img.shields.io/crates/v/deno.svg)](https://crates.io/crates/deno)
 [![Twitter badge][]][Twitter link]
```

File: README.md (hunk 1 above)
Comment (bartlomieju) [reply README.md:81]:
```ts
serve(_req => new Response("Hello, World!"));
```

Comment (iuioiua) [reply README.md:81]: The entire snippet can be:
```ts
import { serve } from "https://deno.land/std@0.182.0/http/server.ts";

serve((_req) => new Response("Hello, World!"));
```
1. By default, `serve()` prints `Listening on http://localhost:8000/` so the `console.log()` isn't needed.
2. Running `deno fmt` adds brackets to `_req` parameter.

File: README.md [hunk 4]
```diff
@@ -62,34 +62,34 @@ and [releases](https://github.com/denoland/deno/releases) for other options.
...
+
+### Additional Resources
 
-The complete API reference is available at the runtime
-[documentation](https://doc.deno.land).
+- **[The Deno Manual](https://deno.land/manual)** is a great starting point for [additional examples](https://deno.land/manual/examples), [setting up your environment](https://deno.land/manual/getting_started/setup_your_environment), [using npm](https://deno.land/manual/node), and more.
+- **[Runtime API reference](https://deno.land/api)** documents all APIs built into Deno CLI.
+- **[Deno Standard Modules](https://deno.land/std)** do not have external dependencies and are reviewed by the Deno core team.
+- **[deno.land/x](https://deno.land/x)** is the registry for third party modules.
+- **[Blog](https://deno.com/blog)** is where the Deno team writes how tos, about solving technical problems, and important product updates.
```
Comment (bartlomieju) [inline README.md:92]:
```suggestion
- **[Blog](https://deno.com/blog)** is where the Deno team writes important product updates and "how to"s, about solving technical problems.
```

**Pull request merged**